executioncontext/projectview/src/test/java/org/jetbrains/bsp/bazel/projectview/parser/DefaultProjectViewParserTest.kt

```diff
@@ -132,6 +128,36 @@ class DefaultProjectViewParserTest {
             projectView.buildFlags shouldBe null
         }
 
+        @Test
+        fun `should return empty directories section for file without directories section`() {
+            // given
+            val projectViewFilePath = Path("/projectview/without/javapath.bazelproject")
```

Comment:
```suggestion
            val projectViewFilePath = Path("/projectview/without/directories.bazelproject")
```

and u have to create thsi file

Reply:
Fixed